lib/_http_client.js

```diff
@@ -65,6 +65,15 @@ function isInvalidPath(s) {
   return false;
 }
 
+function validateHost(host) {
+  // eslint-disable-next-line eqeqeq
```

Comment:
If I'm not wrong this can be removed if `host != null` is used instead of `host != undefined`.